### PR TITLE
Append username to container to avoid stepping on each other feet

### PR DESCRIPTION
**Pull request by dalg24**
I was editing the source on my home dir and the container was pointing to @aprokop's...

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -7,7 +7,10 @@ services:
     environment:
       - TRILINOS_DIR=/scratch/source/trilinos/12.10.1
       - TERM=xterm
-    container_name: dtk_dev
+    # Append user name to the container name.  Speciying a container name
```
Comment (Rombur) [inline docker/docker-compose.yml]: typo Speciying

**Comment by dalg24** [comment]
@Rombur @sslattery @aprokop You all want probably want to update your source and apply that to your containers when this gets merged.

**Comment by dalg24** [comment]
ping

**Comment by aprokop** [comment]
An improvement could also be to allow a user to name the containers. Something like
`CNAME=containername docker-compose up -d`
with 
```pythone
 container_name: dtk_dev_$USER:$CNAME
```
in the docker-compose file.

**Pull request merged**